test/specs/plugin.filler.tests.js

```diff
@@ -223,14 +223,14 @@ describe('Plugin.filler', function() {
 			});
 
 			expect(decodedFillValues(chart)).toEqual([
-				'start', // 'origin'
-				0,       // 1 - 1
-				1,       // 1
-				1,       // 3 - 2
-				5,       // 4 + 1
-				7,       // 5 + 2
-				5,       // 6 - 1
-				'end',   // 'end'
+				'start',	// 'origin'
```

Comment:
I was able to reconfigure the `no-multi-spaces` option to stop creating errors when it's a comment after a line